package.json

```diff
@@ -18,22 +18,24 @@
   },
   "repository": {
     "type": "git",
-    "url": "http://github.com/vogievetsky/dvl.git"
+    "url": "http://github.com/metamx/dvl.git"
   },
-  "main": "./dvl.js",
+  "main": "build/index.js",
```

Comment:
Wondering if there would be a difference specifying this or `dist/dvl.js` as the main entry point

Reply:
The way I see it, `build/index.js` is the entry point if you're requiring the library in JavaScript; `dist/dvl.js` is for using it as a static resource in an HTML `<script>` tag.  The [`main` attribute](https://docs.npmjs.com/files/package.json#main) is for the former.